api/IDBEnvironment.json

```diff
@@ -0,0 +1,110 @@
+{
+  "api": {
+    "IDBEnvironment": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBEnvironment",
+        "support": {
+          "webview_android": {
+            "version_added": "4.4"
```

Comment:
Webview uses Chromium version numbers, not Android numbers. Please change to `"24"`.

Reply:
👍